libs/ledger-live-common/.unimportedrc.json

```diff
@@ -417,6 +417,7 @@
     "src/modularDrawer/hooks/useAssetAccountCounts.tsx",
     "src/modularDrawer/hooks/useSearch.ts",
     "src/modularDrawer/hooks/useModularDrawerConfiguration.ts",
+    "src/modularDrawer/hooks/useCurrenciesUnderFeatureFlag.ts",
```

Comment:
I think this file is imported in several files is this still needed ?

Reply:
Yep because not imported in common directly.. :/